camp_fin/models.py

```diff
@@ -90,13 +90,64 @@ class Campaign(models.Model):
     qual_campaign_id = models.IntegerField(null=True)
     biannual = models.NullBooleanField()
     from_campaign = models.ForeignKey('Campaign', db_constraint=False, null=True)
-    
+    active_race = models.ForeignKey('Race', db_constraint=False, null=True)
+
     def __str__(self):
         candidate_name = '{0} {1}'.format(self.candidate.first_name, 
                                           self.candidate.last_name)
         office = self.office.description
         return '{0} ({1})'.format(candidate_name, office)
 
+    @property
+    def funds_raised(self):
+        filings = self.candidate.entity.filing_set.all()
+        return sum(filing.closing_balance for filing in filings)
```

Comment:
I'm not sure this math is right. Think of a filing like a bank statement. What's the intent here?